packages/fxa-auth-server/bin/key_server.js

```diff
@@ -11,9 +11,11 @@ require('../lib/newrelic')();
 
 const jwtool = require('fxa-jwtool');
 const P = require('../lib/promise');
+const StatsD = require('hot-shots');
 
 function run(config) {
   const log = require('../lib/log')(config.log);
+  const statsd = new StatsD(config.statsd);
```

Comment:
Peeking at the docs, there appears to be [an `errorHandler` that we can set up here](https://www.npmjs.com/package/hot-shots#errors) and trap all errors. Maybe we want to do that here, rather than have the errors bubble up from inside service code?

Reply:
Yeah.  When I was tailing the log, I did not notice any errors when there's nothing listening on the port.  (It's UDP so that kind of make sense.)  But it does throw a DNS error if it cannot resolve the host from the config.